### ISSUE TITLE
ZINC2m IsADirectoryError in utils.get_line_count(path)

### ISSUE THREAD

**Issue by schinto** [DeepGraphLearning/torchdrug#33]
```python
import torch
from torchdrug import core, datasets, tasks, models

dataset = datasets.ZINC2m("~/molecule-datasets/", node_feature="pretrain", edge_feature="pretrain")
```

```
~/opt/anaconda3/envs/drugs/lib/python3.8/site-packages/torchdrug/datasets/zinc2m.py in __init__(self, path, verbose, **kwargs)
     44             reader = csv.reader(fin)
     45             if verbose:
---> 46                 reader = iter(tqdm(reader, "Loading %s" % path, utils.get_line_count(path)))
     47             smiles_list = []
     48 

~/opt/anaconda3/envs/drugs/lib/python3.8/site-packages/torchdrug/utils/file.py in get_line_count(file_name, chunk_size)
    112     """
    113     count = 0
--> 114     with open(file_name, "rb") as fin:
    115         chunk = fin.read(chunk_size)
    116         while chunk:

IsADirectoryError: [Errno 21] Is a directory: '/username/molecule-datasets/'
```

**Comment by KiddoZhu** [comment]
Hi! Thanks for pointing out the bug. We will fix it.

**Comment by Oxer11** [comment]
Fixed in d1428a0
